content/hints.md

```diff
@@ -0,0 +1,127 @@
+
+- For problems with large I/O,
+  it is in your best interest to use buffered input and output in Java.
+  We have tried to make sure that the I/O-intensive problems
+  are solvable using even slow I/O.
+  But for some problems it can be more difficult to get accepted
+  using e.g. the `Scanner` class,
+  since it consumes a significant amount of valuable CPU time
+  that could otherwise be used by your algorithm.
+  In most cases, a `BufferedReader` will be more efficient.
+  Also consider explicit output buffering
+  (e.g., by using the `BufferedWriter` class)
+  to avoid flushing every line of output separately.
+
+- We guarantee that every problem is solvable in C++, Java, and Python 3.
+  There is no such guarantee for Kotlin.
+
+
+## Problem Statements
+
+We generally try to make the input and output formats
+as easy as possible to follow for you as contestants —
+input and output are meant to be the easiest parts of the problems!
+
+- For input, multiple numbers or words on one input line
+  will be separated by single spaces.
+  There are no additional spaces, tabs or newlines.
+
+- For output, unless explicitly stated in the problem statement,
+  we do not require you to output
+  exactly the right number of spaces and newlines.
+  Errors in whitespace within reason will be accepted,
+  but if all whitespace between two tokens are omitted,
+  that is (of course) an error.
+
+- For problems with real-valued output,
+  we generally only require that your output is correct
+  up to a certain absolute or relative error.
+  In the case that both are specified, the largest of the two applies.
+  For example, if the problem statement requests a
+  "absolute or relative error of at most $10^{-6}$", this means that:
+
+  - If the correct answer is $0.005$, any answer between $0.004999$ and $0.005001$ will be accepted.  
+    The absolute error of $\pm 10^{-6}$ is larger than the relative error of $\pm 5 \cdot 10^{-9}$ ($= 0.005 \cdot 10^{-6}$).
+  - If the correct answer is $5000$, any answer between $4999.995$ and $5000.005$ will be accepted.  
+    The relative error of $\pm 0.005$ ($= 5000 \cdot 10^{-6}$) is larger than the absolute error of $\pm 10^{-6}$.
+
+- For problems with real-valued output, any reasonable format for floating point numbers is acceptable.
+  For instance, `17.000000`, `0.17e2`, and `17`
+  are all acceptable ways of formatting the number $17$.
+  For the definition of reasonable, please use your common sense.
+
+- Some problems will state:
+  "If there are multiple valid solutions, you may output any one of them."
+  It may be the case that, on the public sample inputs,
+  your submission provides a different correct output
+  than the corresponding sample output.
+  In that case, you should manually check
+  whether your output conforms to the given output constraints.
+
+
+## Interactive Problems
+
+- The contest typically includes interactive problems.
+  Each interactive problem is accompanied with a Python script
```

Comment:
```suggestion
  Each interactive problem is accompanied by a Python script
```

Reply:
I've learned that "by" is used with persons and "with" is used with things. So I guess "accompanied with a script" is fine?